src/barfturn.ts

```diff
@@ -0,0 +1,284 @@
+      if (steveRoom && canAdventure(steveRoom) && steveRoom !== ghostLocation) {
+        const fightingSteve = steveRoom === $location`The Haunted Laboratory`;
+        // Technically drops 500 meat, but that's close enough for me.
+        if (fightingSteve) embezzlerPrep();
+        const plan = steveAdventures.get(steveRoom);
+        if (plan) {
+          withMacro(
+            Macro.if_($monster`Stephen Spookyraven`, Macro.basicCombat()).abort(),
+            () => {
+              visitUrl(toUrl(steveRoom));
+              for (const choiceValue of plan) {
+                runChoice(choiceValue);
+              }
+              if (fightingSteve || currentRound()) runCombat();
+            },
+            true
+          );
+        }
+        return fightingSteve;
+      }
+      return false;
+    },
+  },
+  {
+    name: "Proton Ghost",
+    available: () =>
+      sober() &&
+      have($item`protonic accelerator pack`) &&
+      get("questPAGhost") !== "unstarted" &&
+      !!get("ghostLocation"),
+    execute: () => {
+      const ghostLocation = get("ghostLocation");
+      if (!ghostLocation) return false;
+      freeFightPrep(
+        new Requirement(
+          ghostLocation === $location`The Icy Peak` ? ["Cold Resistance 5 min"] : [],
+          {
+            forceEquip: $items`protonic accelerator pack`,
+          }
+        )
+      );
+      adventureMacro(ghostLocation, Macro.ghostBustin());
+      return get("questPAGhost") === "unstarted";
+    },
+  },
+  {
+    name: "Vote Wanderer",
+    available: () =>
+      sober() &&
+      have($item`"I Voted!" sticker`) &&
+      totalTurnsPlayed() % 11 === 1 &&
+      get("lastVoteMonsterTurn") < totalTurnsPlayed() &&
+      get("_voteFreeFights") < 3,
+    execute: () => {
+      freeFightPrep(new Requirement([], { forceEquip: $items`"I Voted!" sticker` }));
+      adventureMacroAuto(determineDraggableZoneAndEnsureAccess(), Macro.basicCombat());
+      return get("lastVoteMonsterTurn") === totalTurnsPlayed();
+    },
+  },
+  {
+    name: "Digitize Wanderer",
+    available: () => Counter.get("Digitize Monster") <= 0,
+    execute: () => {
+      const isEmbezzler = get("_sourceTerminalDigitizeMonster") === embezzler;
+      const start = get("_sourceTerminalDigitizeMonsterCount");
+
+      const shouldGoUnderwater =
+        isEmbezzler &&
+        !get("_envyfishEggUsed") &&
+        myLevel() >= 11 &&
+        (getModifier("Adventure Underwater") ||
+          waterBreathingEquipment.some((item) => have(item))) &&
+        (getModifier("Underwater Familiar") ||
+          familiarWaterBreathingEquipment.some((item) => have(item))) &&
+        (have($effect`Fishy`) || (have($item`fishy pipe`) && !get("_fishyPipeUsed"))) &&
+        !have($item`envyfish egg`) &&
+        canAdventure($location`The Briny Deeps`);
+      const targetLocation = shouldGoUnderwater
+        ? $location`The Briny Deeps`
+        : determineDraggableZoneAndEnsureAccess();
```

Comment:
This is some legacy code, but there are players who will encounter so few embezzlers that they won't hit their first digitize during their initial push. It is a pretty niche case, though, that might be worth ceasing to support. Especially because we aren't actually checking if the envyfish egg is worth it.